pages/nfts/new.tsx

```diff
@@ -7,15 +7,17 @@ import Verify from '@/modules/nfts/components/wizard/Verify';
 import InfoScreen from '@/modules/nfts/components/wizard/InfoScreen';
 import { useForm } from 'antd/lib/form/Form';
 import Summary from '@/modules/nfts/components/wizard/Summary';
-import RoyaltiesCreators from '@/modules/nfts/components/wizard/RoyaltiesCreators';
+import RoyaltiesCreators, {
+  HOLAPLEX_CREATOR_OBJECT,
+} from '@/modules/nfts/components/wizard/RoyaltiesCreators';
 import { WalletContext } from '@/modules/wallet';
 import PriceSummary from '@/modules/nfts/components/wizard/PriceSummary';
 import MintInProgress from '@/modules/nfts/components/wizard/MintInProgress';
 import { isNil } from 'ramda';
 import OffRampScreen from '@/modules/nfts/components/wizard/OffRamp';
 import { Connection } from '@solana/web3.js';
 
-export const MAX_CREATOR_LIMIT = 5;
```

Comment:
When is this checked? With holaplex in the creators array is 5 correct?

Reply:
The MAX_CREATOR_LIMIT is used in the Royalties page for the creators array and I changed it to 4 after I removed Holaplex from that creators array